onmt/modules/Generator.lua

```diff
@@ -9,28 +10,67 @@ local Generator, parent = torch.class('onmt.Generator', 'onmt.Network')
 -- for back compatibility - still declare FeaturesGenerator - but no need to define it
 torch.class('onmt.FeaturesGenerator', 'onmt.Generator')
 
+local options = {
+  {
+    '-importance_sampling', false,
```

Comment:
after long discussions - we opted for "-sample_tgt_vocab", and the option is now part of the sampling options